.travis.yml

```diff
@@ -8,8 +8,10 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - RAILS_ENV=test
```

Comment:
Why was this necessary?  I ask because the test:vmdb does this for you, so it *shouldn't* be needed, and if devs don't set this locally, what will happen?